src/support/azext_support/aaz/latest/support/in_subscription/chat_transcript/_show.py

```diff
@@ -182,9 +182,7 @@ def _build_schema_on_200(cls):
             messages.Element = AAZObjectType()
 
             _element = cls._schema_on_200.properties.messages.Element
-            _element.body = AAZStrType(
-                flags={"required": True},
-            )
+            _element.body = AAZStrType()
```

Comment:
Same here, check in other files as well

Reply:
auto generated

Reply:
Its generated based on workspace, check if anything has changed there